langchain/src/callbacks/handlers/tracers.ts

```diff
@@ -378,15 +452,6 @@ export class LangChainTracer extends BaseTracer {
   }
 
   copy(): LangChainTracer {
-    // TODO: this is a hack to get tracing to work with the current backend
-    // we need to not use execution order, then remove this check
-    if (this.executionOrder === 1) {
-      const copy = new LangChainTracer();
-      copy.session = this.session;
-      copy.runMap = new Map(this.runMap);
-      copy.executionOrder = this.executionOrder;
-      return copy;
-    }
     return this;
```

Comment:
This is safe now because `execution_order` gets calculated every time it's accessed?

Reply:
Yes exactly